### ISSUE TITLE
Make Starter Error on WSL2 with Docker Desktop

### ISSUE THREAD

**Issue by sawyerb-ksu** [Islandora-Devops/isle-dc#412]
Issue:
When running `make starter` or `make up` the build is failing with the following error:
`/path/to/secrets/live/SECRET is mounted on / but it is not a shared mount.`

System Setup:
Docker Desktop Version: 4.36.0 (175267)
WSL version: 2.0.14.0

This issue seems to only occur on Docker Desktop.

**Comment by seth-shaw-asu** [comment]
I can confirm that updating Docker Desktop to 4.36.0 creates this issue:
```
Error response from daemon: path /home/sshaw28/isle-dc/secrets/live/JWT_ADMIN_TOKEN is mounted on / but it is not a shared mount.
make[1]: *** [Makefile:687: compose-up] Error 1
make[1]: Leaving directory '/home/sshaw28/isle-dc'
make: *** [Makefile:161: starter] Error 2
```

**Comment by noahwsmith** [comment]
Mac or Windows? I've had to stay on the old virtualization option for Docker For Mac with these most recent updates...

**Comment by seth-shaw-asu** [comment]
Windows (using the Windows Subsystem for Linux).

I've attached the full output of the run; in case it helps anyone figure out what might be going on.
[complete_output.txt](https://github.com/user-attachments/files/18100759/complete_output.txt)

**Comment by DonRichards** [comment]
Try replacing `docker compose exec -T drupal with-contenv bash -lc 'chown -R nginx:nginx /var/www/drupal/ ; su nginx -s /bin/bash -c "composer install"'` with `docker compose exec -T -u nginx drupal sh -c 'composer install'`

**Comment by seth-shaw-asu** [comment]
@DonRichards, nope, same error. Note that it errors out in the `compose-up` which is run before that line.

The command throwing the error is `docker compose up -d --remove-orphans`. Something earlier in the process is causing the issue.

**Comment by seth-shaw-asu** [comment]
Found an issue report documenting the same issue with this version of Docker Desktop: https://github.com/docker/for-win/issues/14434

The issue indicates the problem is resolved by upgrading to 4.37.0. I upgraded to 4.37.1 and confirm this error no longer occurs.

@sawyerb-ksu, can you please also upgrade Docker Desktop and confirm if it resolves this issue for you?

**Comment by sawyerb-ksu** [comment]
Thanks for the find @seth-shaw-asu . That seems to have resolved the issue on my end as well.
